### PR TITLE
Ft/menu

### PR DESCRIPTION
**Pull request by izibyosevictoire**

### CODE REVIEW

**Review by uwumukiza123 — COMMENTED** [review]
I looks good on my sight

**Comment by uwumukiza123** [comment]
Good

**Review by Yvad60 — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
 # git-cafe-exercise
+## bUNDLE 5
+### EXRCISE 2
+``` bash
+victoire@victoire-Latitude-E7440:~/Videos/the gym/git-cafe-exercise$ git add .
+victoire@victoire-Latitude-E7440:~/Videos/the gym/git-cafe-exercise$ git commit -m "changes"
```
Comment (Bateyjosue) [inline README.md:6]: Hey @izibyosevictoire, It's a good practice to provide a clear commit message to let your teammates understand what changes you have worked on, so please  I would suggest you in your future project to use descriptive commit message.

In the meantime, you can refer to the following article https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/  to learn more about  commit messages and how important they are

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+To https://github.com/izibyosevictoire/git-cafe-exercise.git
+   764c72e..4b21b63  main -> main
+```
+
+## Bundle 6
+### exercise 1
+
+``` bash
+victoire@victoire-Latitude-E7440:~/Videos/the gym/git-cafe-exercise$ git add .
+victoire@victoire-Latitude-E7440:~/Videos/the gym/git-cafe-exercise$ git commit -m "menu changes"
```
Comment (Bateyjosue) [inline README.md:26]: Refer to the comment above Please

**Review by Bateyjosue — APPROVED** [review]
Hey @izibyosevictoire, Nice Work so far I have gone through the PR and everything looks pretty well
## Highlight
- [ ] I've made inline comments even thou they are optional, but make sure to remember them in your future work 

### Cheers 🎉 🎉 🎉 

## 
> In case of any questions please don't forget to tag me on @Bateyjosue so that I can be notified
##

**Pull request merged**